Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs

```diff
@@ -75,6 +75,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionBuilder)
         public DbSet<TestAnswers> TestAnswers { get; set; }
         public DbSet<AttendeeAnswers> AttendeeAnswers { get; set; }
         public DbSet<Report> Report { get; set; }
+        public DbSet<TestIpAddress> TestIPAddress { get; set; }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename property 'TestIPAddress' to match camel case naming rules, consider using 'TestIpAddress'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS100)